dotnet/src/Connectors/Connectors.Amazon/Bedrock/Extensions/BedrockKernelBuilderExtensions.cs

```diff
@@ -24,7 +24,7 @@ public static class BedrockKernelBuilderExtensions
     /// <param name="modelId">The model for chat completion.</param>
     /// <param name="bedrockRuntime">The IAmazonBedrockRuntime to run inference using the respective model.</param>
     /// <param name="serviceId">The optional service ID.</param>
-    /// <returns></returns>
+    /// <returns>IKernelBuilder object.</returns>
```

Comment:
When using types try to use the `<see cref>` this helps keep track of the types in the docs.

```suggestion
    /// <returns><see cref="IKernelBuilder"/> object.</returns>
```